.github/docker.yml

```diff
@@ -0,0 +1,48 @@
+name: Build container images for release
+
+on:
+  # Triggers the workflow on new releases
+  release:
+    types:
+      - "created"
+      - "edited"
+      - "published"
+      - "released"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            ghcr.io/${{ github.repository_owner }}/mooncord
+          tags: |
+            type=ref,event=pr
+            type=ref,event=branch
+            type=ref,event=tag
+            type=sha
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to ghcr.io/$
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ github.token }}
```

Comment:
should it be  secrets.GITHUB_TOKEN? instead of github.token?

Reply:
Both will work, `github.token` uses the token from the context while `secrets.GITHUB_TOKEN` will use the secret from the workflow. As GitHub puts it "This is functionally equivalent to the GITHUB_TOKEN secret"

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context